util/process.go

```diff
@@ -38,19 +39,40 @@ func ExecAndCaptureOutput(command string, args ...string) (string, string, error
 	subcmd.Stdout = wStdout
 	subcmd.Stderr = wStderr
 
-	err = subcmd.Run()
+	outputStdout := make([]byte, 0, 10240)
+	outputStderr := make([]byte, 0, 10240)
+	done := make(chan struct{})
 
-	wStdout.Close()
-	outputStdout, errIO := ioutil.ReadAll(rStdout)
-	if err == nil {
-		err = errIO
+	var wg sync.WaitGroup
+	wg.Add(2)
+
+	reader := func(input *os.File, output *[]byte) {
+		completed := false
+		for !completed {
+			select {
+			case <-done:
+				completed = true
```

Comment:
with the defer mentioned below, you should be able to return here instead of having this varaible.

Reply:
No, read still needs to be done to read rest of the data